tipocket-ctl/tpctl/params.py

```diff
@@ -24,7 +24,7 @@ def parse_params(params):
         if key in IGNORE_OPTION_LIST:
             continue
         # convert True/False to 'true/false'
-        if key == 'purge':
+        if key in ('purge', 'delNS'):
```

Comment:
Is `delNS` a new feature of tipocket?

Reply:
No. It already exists.
I found that 
* option `purge=true` means tipocket will delete the tidb cluster if test case passed.
  NOTE: it will not delete `pvc`.
* option `delNS=true` means tipocket will delete the whole namespace if test case passed.